src/Win32Util.cpp

```diff
@@ -91,3 +91,86 @@ argv_to_string(const char* const* argv, const std::string& prefix)
 }
 
 } // namespace Win32Util
+
+// From: https://stackoverflow.com/a/26085827/262458
```

Comment:
Why not the shorter more modern looking version a few posts down?
https://stackoverflow.com/a/58162122/12271052